js/script.js

```diff
@@ -17,15 +21,13 @@
   });
 
   app.service("TaskResource", function($resource){
-    return $resource("/tasks/:id", {id:"@id"}, {
+    return $resource("/api/v1/item/:id/", {id:"@id"}, {
```

Comment:
Could you please make a constants class and put `/api/v1` as `BASE` and `/item/` as `ITEM`?